components/survey/Card/index.jsx

```diff
@@ -0,0 +1,47 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import Image from 'next/image';
+
+import angleRightIcon from '@/public/angle-right.svg';
+
+export const surveyCardTestIds = {
+  base: 'survey-card',
+  coverImageUrl: 'survey-card__cover_image_url',
+  title: 'survey-card__title',
+  description: 'survey-card__description',
+  angleRightIcon: 'survey-card__details',
+};
+
+const SurveyCard = ({ slideId, title, description, coverImageUrl }) => (
+  <div data-test-id={surveyCardTestIds.base}>
+    <div className="relative w-full h-[302px] mb-8" data-test-id={surveyCardTestIds.coverImageUrl}>
+      <Image className="rounded-xl" src={`${coverImageUrl}l`} alt="cover image" layout="fill" />
```

Comment:
```suggestion
      <Image className="rounded-xl object-cover" src={`${coverImageUrl}l`} alt="cover image" layout="fill" />
```

I just realized that the ratio of cover images was broken (so the pictures were compressed or expanded**).
To prevent that, while keeping the rounded corners, we can use `object-fit: cover;` in css (the code suggestion uses the TailWind syntax). 

** Visual is often better than words:

|Compressed|Expanded|
|---|---|
|<img width="175" alt="image" src="https://user-images.githubusercontent.com/77609814/165442991-dbeb035d-34a3-4355-907e-98f8ce15926e.png">|<img width="930" alt="image" src="https://user-images.githubusercontent.com/77609814/165442849-f9cb8787-5577-4755-bee4-b96dce23e11a.png">|

Reply:
Suggestion render:

|Instead of Compressed | Instead of Expanded|
|---|---|
|<img width="173" alt="image" src="https://user-images.githubusercontent.com/77609814/165443157-98c7d5df-807f-408b-b32e-a593661b5724.png">|<img width="827" alt="image" src="https://user-images.githubusercontent.com/77609814/165443258-80c9d05c-63a2-4a0f-aa5f-b5f91f42135e.png">|


Of course these are extreme cases and you might also want to define "limits" (e.g. max-width — min-width can be assume for a screen under 320px)

Reply:
Thanks for the suggestion! `next/image` seems to have a `objectFit` prop exactly for this purpose, added in 664dcd34571dbbbd6d26d745024c495892f75226